plugins/modules/user.py

```diff
@@ -33,13 +33,11 @@
   state:
     description:
     - Specifies the action to be performed for the user.
-    - C(present) creates a user with provided I(name) and I(attributes) in the system.
-    - If the user already exists in the system, the I(attributes) will be changed.
+    - C(present) specifies to create a user if it does not exist, otherwise it changes the
```

Comment:
C(present) creates a user with provided I(name) and I(attributes) in the system. If already present, it can be used to modify the attributes.